src/resources/js/views/scripts.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since  TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events = tribe.events || {};
+tribe.events.views = tribe.events.views || {};
+
+/**
+ * Configures Views Object in the Global Tribe variable
+ *
+ * @since  TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events.views.scripts = {};
+
+/**
+ * Initializes in a Strict env the code that manages the Event Views
+ *
+ * @since  TBD
+ *
+ * @param  {PlainObject} $   jQuery
+ * @param  {PlainObject} _   Underscore.js
+ * @param  {PlainObject} obj tribe.events.views.scripts
+ *
+ * @return {void}
+ */
+( function( $, _, obj ) {
+	'use strict';
+	var $document = $( document );
+
+	/**
+	 * Selectors used for configuration and setup
+	 *
+	 * @since TBD
+	 *
+	 * @type {PlainObject}
+	 */
+	obj.selectors = {
+		tooltip: '.tribe-events-tooltip',
+		tooltipContent: '.tribe-events-tooltip__content',
+	};
+
+	/**
+	 * Initialize accessible tooltips via tooltipster
+	 *
+	 * @since TBD
+	 *
+	 */
+	obj.initTooltips = function() {
+
+		$( obj.selectors.tooltip ).tooltipster( {
+			interactive: true,
+			theme: [ 'tribe-common', 'tribe-events', 'tribe-events-tooltip-theme' ],
+			functionInit: function( instance, helper ) {
```

Comment:
Move this method to the `obj`, so it makes it easier to extend this code.

Reply:
this is a method provided by `tooltipster`, and it needs to be declared there.

Reply:
As we discussed on hangout, we should still move it to the exposed `obj` as a method, so there is an easier way to extend this.